src/foam/blob/Blob.js

```diff
@@ -848,7 +848,7 @@ foam.CLASS({
       class: 'String',
       name: 'address',
       factory: function() {
-        return window.location.origin + '/' + this.serviceName;
+        return globalThis.location.origin + '/' + this.serviceName;
```

Comment:
GlobalThis is meant to abstract away the difference between node.js and the browser, but things like the window.location are only going to be in the browser anyway, so no reason to not just use globalThis.

Reply:
revert this line.